subsys/bluetooth/controller/ll_sw/ull_peripheral_iso.c

```diff
@@ -96,11 +96,21 @@ static struct ll_conn *ll_cis_get_acl_awaiting_reply(uint16_t handle, uint8_t *e
 uint8_t ll_cis_accept(uint16_t handle)
 {
 	uint8_t status = BT_HCI_ERR_SUCCESS;
-	struct ll_conn *acl_conn = ll_cis_get_acl_awaiting_reply(handle, &status);
+	struct ll_conn *conn = ll_cis_get_acl_awaiting_reply(handle, &status);
+
+	if (conn) {
+		uint32_t cis_offset_min;
+
+#if defined(CONFIG_BT_CTLR_JIT_SCHEDULING)
+		cis_offset_min = MAX(400, EVENT_OVERHEAD_CIS_SETUP_US);
```

Comment:
Magic number 400 :) Please describe or define the value

Reply:
@mtpr-ot  400 ? what should I define this as?

Reply:
> Magic number 400 :) Please describe or define the value

This is from the spec, as far as I remember.